src/script/util/Environment.ts

```diff
@@ -28,25 +28,23 @@ const APP_ENV = {
 
 const getAppVersion = (): string => {
   const versionElement = document.head.querySelector("[property='wire:version']");
-  const hasVersion = versionElement && versionElement.hasAttribute('version');
-  return hasVersion ? versionElement.getAttribute('version').trim() : '';
+  return versionElement?.getAttribute('version').trim() || '';
 };
 
 const getElectronVersion = (userAgent: string): string => {
-  // [match, app, version]
-  const [, , electronVersion] = /(Wire|WireInternal)\/(\S+)/.exec(userAgent) || [];
+  // [match, version]
+  const [, electronVersion] = /Wire(?:Internal)?\/(\S+)/.exec(userAgent) || [];
   return electronVersion;
 };
 
 const getFormattedAppVersion = (): string => {
-  const [year, month, day, hour, minute] = getAppVersion().split('-');
-  return `${year}.${month}.${day}.${hour}${minute}`;
+  const [year, month, day, hour, minute] = getAppVersion().split('.');
+  return `${year}.${month}.${day}.${hour}.${minute}`;
```

Comment:
This is the format we are using everywhere now. Should we use it here, too?

Reply:
on webapp we do 
`Config.getConfig().VERSION` which gives -> 2020.09.07.22.34